LoggingKata/TacoParser.cs

```diff
@@ -7,12 +7,67 @@ public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
         
+        
         public ITrackable Parse(string line)
         {
             logger.LogInfo("Begin parsing");
 
-            // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            if (line == null)
+            {
+                return null;
+
+            }
+               
+            var cells = line.Split(',');
+
+             if (cells.Length < 3)
+            {
+                //Log that and return null
+                logger.LogWarning("Not enough data to parse");
+                return null;
+            }
+
+
+            double latitude;
+            double longitude;
+
+            try
+            {
+
+                latitude = double.Parse(cells[0]);
+                 longitude = double.Parse( cells[1]);
+            }
+
+
+                 
```

Comment:
The empty lines between the `try` and `catch` should be removed- there's nothing that should go here to separate the two blocks